app/_hub/kong-inc/openid-connect/how-to/_demonstrating_proof_of_possession.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Demonstrating Proof-of-Possession
+nav_title: Demonstrating Proof-of-Possession
+minimum_version: 3.7.x
+---
+
+## Overview
+
+Demonstrating Proof-of-Possession(DPoP) is an alternative technique to the Mutual TLS certificate-bound access tokens. Instead of binding the token with the client certificate used in the mTLS, it binds the token with a JWK provided by the client.
```

Comment:
```suggestion
Demonstrating Proof-of-Possession (DPoP) is an alternative technique to the [Mutual TLS certificate-bound access tokens](/hub/kong-inc/openid-connect/how-to/client-authentication/mtls/). Instead of binding the token with the client certificate used in the mTLS, it binds the token with a JWK provided by the client.
```